test/topology_custom/test_streaming.py

```diff
@@ -0,0 +1,11 @@
+from test.pylib.manager_client import ManagerClient
+import pytest
+
+@pytest.mark.asyncio
+async def test_drop_table_during_bootstrap(manager: ManagerClient):
+    servers = [await manager.server_add(config={
+        'error_injections_at_startup': ['stream_mutation_fragments_table_dropped'],
+        'enable_repair_based_node_ops': False,
+        'enable_user_defined_functions': False,
```

Comment:
Why do you set enable_user_defined_functions to false explicitly?

Reply:
You can work around this by also specifying `experimental_features` here. I think the default (in `test/pylib/scylla_cluster.py`) adds `udf` to `--experimental-features`, that is why you see this error. But this is not critical to solve, the above also works.

Reply:
Indeed. I didn't see it at first. I know `scylla_cluster.py` uses command line params, maybe they take precedence over the config file here. @kbr-scylla is this `config` param merged with the default command-line params?

Reply:
scylla_cluster.py generates both a scylla.yaml file and passes some commandline parameters. `config` is merged into the scylla.yaml file.

(defaults for scylla.yaml are defined in `make_scylla_conf` function)